shop/flyway.proprieties

```diff
@@ -0,0 +1,4 @@
+flyway.locations=filesystem:C:\SpringTraining\online-shop-alxcl12\shop\src\main\resources\db\migrations
```

Comment:
Please never specify the path like this. Instead use this: filesystem:src/main/resources/db/migration

Reply:
Fixed